tctl-snap/README.md

```diff
@@ -79,3 +92,10 @@ tctl commands can be run in the following pre-defined three environments:
   ```
 
   Other commands can be found [here](https://docs.temporal.io/tctl-v1).
+
+  Note: the --tls-server-name flag must be included if TLS is enabled on your
+  deployment through ingress. To avoid having to include the `address` and
```

Comment:
The flag is `address`, the placeholder value seen above is `<server_hostname>`. They are not set from the beginning because the server address depends on your deployment. As this is meant to be a generic snap, any user should be able to use it to connect to their Temporal cluster by configuring the required parameters.